### DISCUSSION TITLE
Getting Error while installing pgmoneta on Ubuntu 20.04.4 LTS

### DISCUSSION THREAD

**Discussion by sourav014** [pgmoneta/pgmoneta#71] [Q&A]
Hello Everyone,

I have followed these steps as mentioned in the Readme.md to install the pgmoenta in the local.

git clone https://github.com/pgmoneta/pgmoneta.git
cd pgmoneta
mkdir build
cd build
cmake -DCMAKE_INSTALL_PREFIX=/usr/local ..
make
sudo make install

But after doing all the steps, while running the command pgmoneta, i am getting this as error:
pgmoneta: error while loading shared libraries: libpgmoneta.so.0: cannot open shared object file: No such file or directory

**Comment by jesperpedersen** [comment]
You likely need `/usr/local/lib` in your `ld.so.conf` file.

See https://github.com/pgmoneta/pgmoneta/blob/main/doc/tutorial/01_install.md for a tutorial

Hope this helps !
